src/aks-preview/azext_aks_preview/_params.py

```diff
@@ -914,6 +915,10 @@ def load_arguments(self, _):
         c.argument('root_cert_object_name')
         c.argument('cert_chain_object_name')
 
+    with self.argument_context('aks ai') as c:
+        c.argument('prompt', options_list=['--prompt', '-p'], validator=validate_prompt_input,
+                   help='The question you want to ask, e.g: How to create a AKS cluster')
```

Comment:
Using general name instead of professional terminology may be more user-friendly, especially for some customers who may not be familiar with ChatGPT related fields

```suggestion
        c.argument('question', options_list=['--question', '-q'], validator=validate_prompt_input,
                   help='The question you want to ask, e.g: How to create a AKS cluster')
```